components/availability/src/Availability.svelte

```diff
@@ -638,7 +444,10 @@
           numFreeCalendars === 0 ||
           fewerCalendarsThanRatio ||
           (_this.required_participants.length &&
-            !allRequiredParticipantsIncluded(epoch[0].available_calendars))
+            !arrayContainsArray(
```

Comment:
`arrayContainsArray` is not clear as to what is being done here. I'd suggest adding a comment.